src/main/scala/epic/dense/AffineTransform.scala

```diff
@@ -6,7 +6,7 @@ import epic.features.SegmentedIndex
 import epic.framework.Feature
 
 import scala.runtime.ScalaRunTime
-
+import scala.util.Random
```

Comment:
you should probably leave as is for replicability, but in general prefer breeze's Rand generators (using implicit RandBasis for threading through the rng)